src/nft.vy

```diff
@@ -243,7 +248,7 @@ event RoleMinterChanged:
 
 @external
 @payable
-def __init__(name_: String[25], symbol_: String[5], base_uri_: String[80], name_eip712_: String[50], version_eip712_: String[20]):
+def __init__(name_: String[25], symbol_: String[5], base_uri_: String[80], name_eip712_: String[50], version_eip712_: String[20], auction_contract_address_: address):
```

Comment:
Make sure firebase metadata uri will be compatible here!

Reply:
ack: